probes/webhooksWithoutTokenAuth/def.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: webhooksWithoutTokenAuth
+short: This check determines whether webhooks defined in the repository use a token to authenticate to remote servers.
+motivation: >
+  Webhooks without token authorization have the potential to make projects accessible to third-parties.
+implementation: >
+  The probe checks all webhooks of a project and checks whether each uses token authentication.
+outcome:
+  - If the project has one or more webhooks without token authorization, the probe returns as many OutcomeNegative (0) as the project has webhooks without token authorization.
+  - If the project does not have any webhooks without token authorization, the probe returns one OutcomePositive (1).
+remediation:
+  effort: Low
+  text:
+    - Check whether your service supports token authentication.
```

Comment:
typically folks who create webhook don't own the service. SO maybe "Check if the service your webhooks is configured with supports secrets".
Same comment for `markdown` field

Reply:
i think there's a similar phrasing in the checks.md, so maybe look at that?

Reply:
Done in https://github.com/ossf/scorecard/pull/3522/commits/3c03ddadc5105ada45f234b13a56c99ddb2bd8f4